<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelBy">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>581</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Dynamic Memory Allocation Simulator by Jo達o H端tner&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Dynamic Memory Allocation Simulator by Jo達o H端tner</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSystemMemorySize">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>System memory size (mb)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputSystemSize">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNumberOfProcess">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Number of process (max 9)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputNumberProcess">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonProximo">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>560</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Proximo</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelProcess_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Process 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelProcess_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Process 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSize_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Size (mb)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSize_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Size (mb)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Time (t)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Time (t)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelProcess_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Process 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSize_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Size (mb)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Time (t)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputSize_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputTime_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputSize_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputTime_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelProcess_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Process 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSize_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Size (mb)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputSize_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputTime_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputSize_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Time (t)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputTime_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelProcess_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Process 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSize_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Size (mb)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputSize_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Time (t)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputTime_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>350</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelProcess_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Process 6</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSize_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>420</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Size (mb)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputSize_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>450</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>490</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Time (t)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputTime_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>520</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelProcess_7">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Process 7</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSize_7">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Size (mb)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputSize_7">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime_7">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Time (t)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputTime_7">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelProcess_8">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Process 8</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSize_8">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>250</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Size (mb)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputSize_8">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>280</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime_8">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>320</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Time (t)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputTime_8">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>350</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelProcess_9">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>390</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Process 9</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSize_9">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>420</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Size (mb)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputSize_9">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>450</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime_9">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>490</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Time (t)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputTime_9">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>520</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonProximo_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>560</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Proximo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSair">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
   <zorder>labelBy</zorder>
   <zorder>labelProcess_1</zorder>
   <zorder>labelProcess_2</zorder>
   <zorder>labelSize_1</zorder>
   <zorder>labelSize_2</zorder>
   <zorder>labelTime_1</zorder>
   <zorder>labelTime_2</zorder>
   <zorder>labelProcess_3</zorder>
   <zorder>labelSize_3</zorder>
   <zorder>labelTime_3</zorder>
   <zorder>inputSize_1</zorder>
   <zorder>inputTime_1</zorder>
   <zorder>inputSize_2</zorder>
   <zorder>inputTime_2</zorder>
   <zorder>labelProcess_4</zorder>
   <zorder>labelSize_4</zorder>
   <zorder>inputSize_3</zorder>
   <zorder>inputTime_3</zorder>
   <zorder>inputSize_4</zorder>
   <zorder>labelTime_4</zorder>
   <zorder>inputTime_4</zorder>
   <zorder>labelProcess_5</zorder>
   <zorder>labelSize_5</zorder>
   <zorder>inputSize_5</zorder>
   <zorder>labelTime_5</zorder>
   <zorder>inputTime_5</zorder>
   <zorder>labelProcess_6</zorder>
   <zorder>labelSize_6</zorder>
   <zorder>inputSize_6</zorder>
   <zorder>labelTime_6</zorder>
   <zorder>inputTime_6</zorder>
   <zorder>labelProcess_7</zorder>
   <zorder>labelSize_7</zorder>
   <zorder>inputSize_7</zorder>
   <zorder>labelTime_7</zorder>
   <zorder>inputTime_7</zorder>
   <zorder>labelProcess_8</zorder>
   <zorder>labelSize_8</zorder>
   <zorder>inputSize_8</zorder>
   <zorder>labelTime_8</zorder>
   <zorder>inputTime_8</zorder>
   <zorder>labelProcess_9</zorder>
   <zorder>labelSize_9</zorder>
   <zorder>inputSize_9</zorder>
   <zorder>labelTime_9</zorder>
   <zorder>inputTime_9</zorder>
   <zorder>labelNumberOfProcess</zorder>
   <zorder>inputNumberProcess</zorder>
   <zorder>inputSystemSize</zorder>
   <zorder>labelSystemMemorySize</zorder>
   <zorder>buttonProximo</zorder>
   <zorder>buttonProximo_2</zorder>
   <zorder>buttonSair</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>683</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonSair</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>592</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>598</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
